index.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Juliana DeCarvalho Sutherland, Software Engineer Student + Full Stack Developer</title>
+    <link href="https://fonts.googleapis.com/css?family=Caveat|Montserrat:400,600&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="css/styles.css">
+  </head>
+  <body>
+    <main>
+      <!-- ***************  ABOUT / PROFILE  **************** -->
+      <header>
+        <div class="content-wrap">
+          <h1>Juliana DeCarvalho Sutherland</h1>
+          <h2>Software Engineer Student & Environmental Scientist</h2>
+
+          <p>Full-stack developer with a love for new technology, a curious mind for continuous learning, and a passion for solving complex problems in a creative way. I value diversity, inclusion, precision, and attention to detail. In my previous life, I worked with scientists across the globe to create open-source science and scientific materials accessible to all. This allowed me to build a strong foundation in collaboration, relationships, and communication skills.</p>
+          <p>I’m also exploring more creative pursuits with my photography & the love for the outdoors. <a href="https://www.instagram.com/j.by.the.bay/" target="_blank">j.by.the.bay</a></p>
```

Comment:
LOVE the look of your bio with the bear background image 🐻❤️

It is a little hard to read the text so adding a background overlay (aka background color with opacity) would help the text contrast more and make it easier to read. 

So you could surround the text with a div and a class like so:
```suggestion

 <div class="bio">
              <p>Full-stack developer with a love for new technology, a curious mind for continuous learning, and a passion for solving complex problems in a creative way. I value diversity, inclusion, precision, and attention to detail. In my previous life, I worked with scientists across the globe to create open-source science and scientific materials accessible to all. This allowed me to build a strong foundation in collaboration, relationships, and communication skills.</p>
              <p>I’m also exploring more creative pursuits with my photography & the love for the outdoors. <a href="https://www.instagram.com/j.by.the.bay/" target="_blank">j.by.the.bay</a></p>
          </div>
```

Then add the following css:

```
.bio {
  width: 100%;
  margin: 0 1% 0 1%;
  padding: 1rem;
  height: 13rem;
  background-color: rgba(0,0,0,0.5);
} 
 ```

Would create the following effect:

![Screen Shot 2021-06-29 at 12 23 59 PM](https://user-images.githubusercontent.com/16619004/123855581-eb765800-d8d4-11eb-990c-ee268b9f2041.png)